packages/components/src/ButtonDismiss/ButtonDismiss.tsx

```diff
@@ -5,9 +5,14 @@ import { Button } from "../Button";
 interface ButtonDismissProps {
   onClick?(): void;
   ariaLabel: string;
+  size?: "small" | "base";
 }
 
-export function ButtonDismiss({ onClick, ariaLabel }: ButtonDismissProps) {
+export function ButtonDismiss({
+  onClick,
+  ariaLabel,
+  size = "base",
+}: ButtonDismissProps) {
   return (
     <div className={styles.closeButtonWrapper}>
```

Comment:
This is not you, but i'm making a note of it here

We might have to add a `display:flex` on that `closeButtonWrapper` class as it is housing an `inline-*` element. which adds some super nice white space/odd spacing

height of the button is 25.63px

![image](https://user-images.githubusercontent.com/15986172/131544644-dc5bcb26-c442-4d87-bff2-11eb09504621.png)

which makes the height of the chip 31.63

![image](https://user-images.githubusercontent.com/15986172/131544788-4ac83546-5452-4b33-b474-a1ded0a81fa7.png)

Adding the `display: flex` (there might be a different way) evens those babies out

![image](https://user-images.githubusercontent.com/15986172/131544978-e0580128-415d-4e76-a62b-7dfe7c92a22a.png)
![image](https://user-images.githubusercontent.com/15986172/131545019-92233ae8-778d-46e3-ab27-b72855267cd8.png)

Which is, I'm going to assume, the height that we want(??) and wouldn't mess with some vertical alignment things.

And most importantly, fix this alignment issue as well

Before
![image](https://user-images.githubusercontent.com/15986172/131545915-45de7efc-ed46-4b11-82ab-c9adb48a39e5.png)

After
![Chip___🔱_Atlantis](https://user-images.githubusercontent.com/15986172/131546136-1616ea89-2b50-4f8e-ac01-2c4ecba5e2ef.png)

Reply:
Would we be able to move this discussion to an issue regarding to the dismiss button?

Reply:
Yup